openapi.yaml

```diff
@@ -6918,6 +6974,125 @@ paths:
       - lang: CLI
         source: >
           linode-cli lke pool-recycle 12345 456
+  /lke/clusters/{clusterID}/nodes/{nodeId}:
+    parameters:
+    - name: clusterId
+      in: path
+      description: ID of the Kubernetes cluster containing the Node.
+      required: true
+      schema:
+        type: integer
+    - name: nodeId
+      in: path
+      description: ID of the Node to look up.
+      required: true
+      schema:
+        type: string
+    x-linode-cli-command: lke
+    get:
+      operationId: getLKEClusterNode
+      x-linode-cli-action: node-view
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - lke:read_write
+      tags:
+      - Linode Kubernetes Engine (LKE)
+      summary: Node View
+      description: >
+        Returns the values for a specified node object.
+      responses:
+        '200':
+          description: Returns the values of a node object in the form that it appears currently in the node pool array.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: object
+                    description: >
+                      The selected node in the cluster.
+                    properties:
+                      id:
+                        type: string
+                        readOnly: true
+                        description: >
+                          The Node's ID.
+                        example: "12345-6aa78910bc"
+                      instance_id:
+                        type: integer
+                        description: >
+                          The Linode's ID. If no Linode is currently provisioned for this Node, this is `null`.
+                        example: 123456
+                      status:
+                        type: string
+                        description: >
+                          The Node's status as it pertains to being a Kubernetes node.
```

Comment:
This status relates more to whether or not the Linode has successfully been created with the necessary Kubernetes software. `not_ready` means that the Linode is still in the process of being created, while `ready` means that the Linode has all the software and is in a Running state. It does _not_ mean that the Kubernetes Node Object (which you can see when you `kubectl get nodes`) itself is ready. We want to add that information and have it be accessible from APIv4 at some point in the future.